types/fparser/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for fparser 2.0
+// Project: https://github.com/bylexus/fparse
+// Definitions by: bylexus <https://github.com/bylexus>,
+//                 andrew0687 <https://github.com/andrew0687>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class Formula {
+    constructor(fStr: string, options?: object);
+    setFormula(formulaString: string): this;
+    enableMemoization(): void;
+    disableMemoization(): void;
+    splitFunctionParams(toSplit: string): string[];
+    cleanupInputString(s: string): string;
+    parse(str: string): any;
+    private _do_parse(str: string): any;
```

Comment:
```suggestion
```
is this for internal use or for consumption in other code? If the last one, do not use `private`